src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/sparkrecords/Pair.java

```diff
@@ -55,21 +55,36 @@ public Pair(final GATKRead read1, final GATKRead read2, final SAMFileHeader head
         final int read1UnclippedStart = ReadUtils.getStrandedUnclippedStart(read1);
         final int read2UnclippedStart = ReadUtils.getStrandedUnclippedStart(read2);
 
-        if( read1UnclippedStart < read2UnclippedStart ){
+        int read1ReferenceIndex = ReadUtils.getReferenceIndex(read1,header);
+        int read2ReferenceIndex = ReadUtils.getReferenceIndex(read2,header);
+
+        if( read1ReferenceIndex != read2ReferenceIndex ? read1ReferenceIndex < read2ReferenceIndex : read1UnclippedStart <= read2UnclippedStart ){
```

Comment:
Wouldn't this would be easier to read as 
```
if( read1ReferenceIndex <= read2ReferenceIndex && read1UnclippedStart <= read2UnclippedStart) {
```

unless I missed something there that makes them not equivalent

Reply:
Oh, heh, you're write.  I didn't think that through.